### PR TITLE
Fix kwarg injection default values

### PR DESCRIPTION
**Pull request by vachillo**
- [x] I have read and agree to the contributing guidelines for [submitting new pull requests](https://github.com/griptape-ai/griptape?tab=readme-ov-file#submitting-pull-requests).

## Describe your changes
the activity decorator currently overwrites default values with `None` if they are not passed to the activity.
context from below:

currently if you defined a schema with 3 params: `foo` , `bar`, `baz`, and your activity method was defined as:
```
@activity(...)
def activity_method(foo: str, bar: str, baz: str = "default"): ...
```
and the input looked like:
```
{
  "values": {
    "foo": "some_foo",
    "bar": "some_bar"
  },
}
```
the decorator would set `baz=None` rather than keeping the default that was defined on the method signature

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `MarqoVectorStoreDriver.query` failing when `include_metadata` is `True`.
 - `with_contextvars` not properly wrapping functions in some cases.
 - Crash when calling `ToolkitTask.run()` directly.
+- Support default values using kwarg injection on `@activity` methods.
```
Comment (collindutter) [inline CHANGELOG.md:96]: Is this meant to be under Fixed?

**Review by collindutter — COMMENTED** [review]
Can you provide some context in the PR description?

File: CHANGELOG.md (hunk 1 above)
Comment (vachillo) [reply CHANGELOG.md:96]: I think so. currently if you defined a schema with 3 params: `foo` , `bar`, `baz`, and your activity method was defined as:
```
def activity_method(foo: str, bar: str, baz: str = "default"): ...
```
and the input looked like:
```
{
  "values": {
    "foo": "some_foo",
    "bar": "some_bar"
  },
}
```
the decorator would set `baz=None` rather than keeping the default that was defined on the method signature

Comment (collindutter) [reply CHANGELOG.md:96]: Got it. Can you rephrase the changelog entry with an implicit "Fixed"?

**Review by collindutter — APPROVED** [review]
